Applications/ApplicationsLib/ProjectData.cpp

```diff
@@ -338,6 +339,13 @@ void ProjectData::parseProcesses(BaseLib::ConfigTree const& processes_config,
                         "dimension");
             }
         }
+        else if (type == "DensityDrivenFlow")
```

Comment:
Just discussed with Fabien: Thermohaline would be fine if we had mass transport included. At this stage we should first decide how to handle the processes. Should we implement one process with the possibility to switch between thermal convection and thermaohaline convection or should there be two implementations?